util/test/annotate.py

```diff
@@ -45,10 +45,10 @@ def load(path):
   return data
 
 
-def get(data, graph, series, start, end):
+def get(data, graph, series, start, end, hostname=_hostname):
```

Comment:
Perhaps this shouldn't be called `hostname` anymore? I'd be tempted to remove the default argument for it as well since I don't think we will ever use it.